src/webview/MessageList.js

```diff
@@ -176,10 +180,11 @@ class MessageListInner extends React.Component<Props> {
     const contentHtml = messageListElementsForShownMessages
       .map(element => messageListElementHtml({ backgroundData, element, _ }))
       .join('');
-    const { auth, theme } = backgroundData;
+    const { auth } = backgroundData;
+
     const html: string = getHtml(
```

Comment:
nit: no new blank line here, as it doesn't correspond to a boundary between logical groups

(If anything, one wants to think of `contentHtml` above and `html` here as grouped together, and `auth` destructured from `backgroundData` as grouped with the object-destructuring from `props` above.)